spec/views/rb_taskboards/show_spec.rb

```diff
@@ -35,60 +35,69 @@
 
 require File.dirname(__FILE__) + '/../../spec_helper'
 
-describe 'rb_taskboards/show', :type => :view do
+describe 'rb_taskboards/show', type: :view do
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.